Sources/System/Internals/Exports.swift

```diff
@@ -7,19 +7,19 @@
  See https://swift.org/LICENSE.txt for license information
 */
 
-@_implementationOnly import CSystem
+import CSystem
```

Comment:
We'll need to keep the `CSystem` import `@_implementationOnly`, unless we're willing to make it public. (Which I don't think we would want to do in the ABI-stable incarnation of System.)

Reply:
Err, so how would that work? Linux gets it definitions from `CSystem`, not `GlibC`.

Reply:
If we need to have `System` as a public import on Linux, then we should definitely import it there.

All I'm saying is that on Darwin, `CSystem` should only be imported `@_implementationOnly` (or not at all, given that we don't seem to need it).

Reply:
That will work until we do need it, at least. We wouldn't be able to surface `ResourceUsageInfo` as a raw-valued struct, but that's probably fine for the foreseeable future.